### PR TITLE
Psp-4735 lease updates

### PR DESCRIPTION
**Pull request by devinleighsmith**
Allow updates of lease pages on map side bar. Also allows update of core lease details page. Refactors full/half screen display logic to use existing contexts.

### CODE REVIEW

**Comment by devinleighsmith** [comment]
@asanchezr If you haven't gotten too far with your work on lease details update you may just want to start with this.

File: source/frontend/src/features/documents/list/DocumentListContainer.tsx [hunk 1]
```diff
@@ -79,6 +79,7 @@ const DocumentListContainer: React.FunctionComponent<
       setStaleFile(true);
     }
   }, [file, setStaleFile, retrieveDocuments]);
+  console.log(props.disableAdd);
```
Comment (asanchezr) [inline source/frontend/src/features/documents/list/DocumentListContainer.tsx:82]: console.log

File: source/frontend/src/features/documents/list/DocumentListView.tsx [hunk 2]
```diff
@@ -156,6 +156,7 @@ export const DocumentListView: React.FunctionComponent<
   };
 
   const getHeader = () => {
+    console.log(props.disableAdd);
```
Comment (asanchezr) [inline source/frontend/src/features/documents/list/DocumentListView.tsx:159]: console.log

**Comment by devinleighsmith** [comment]
@asanchezr anything else? I'd like to address any comments in a single commit if possible.

File: source/frontend/src/features/leases/detail/LeasePages/details/UpdateLeaseContainer.tsx [hunk 3]
```diff
@@ -63,17 +65,18 @@ export const UpdateLeaseContainer: React.FunctionComponent<
...
+      onEdit(false);
     }
   };
 
+  const initialValues = LeaseFormModel.fromApi(apiLease);
   return (
     <>
       <LoadingBackdrop show={loading} parentScreen></LoadingBackdrop>
       <UpdateLeaseForm
         onCancel={() => history.push(`/lease/${apiLease?.id}`)}
```
Comment (asanchezr) [inline source/frontend/src/features/leases/detail/LeasePages/details/UpdateLeaseContainer.tsx:77]: wouldn't this should redirect to the sidebar URL for leases?

**Review by asanchezr — APPROVED** [review]

**Pull request merged**